lib/manager/actions/user.js

```diff
@@ -273,6 +273,9 @@ export function updateUserData (user: any, userData: any) {
         } else {
           dispatch(fetchUsers())
         }
+        // We need to update directly localStorage as Auth0 returns stale data
+        // see https://github.com/ibi-group/datatools-ui/issues/452
+        window.localStorage.setItem('profile', JSON.stringify(profile))
```

Comment:
It'd probably be best to use `Auth0Manager.setProfile` to avoid duplicating code.